Formula/s/synergy-core.rb

```diff
@@ -66,14 +71,20 @@ class SynergyCore < Formula
 
   def install
     # Use the standard brew installation path.
-    inreplace "CMakeLists.txt",
-              "set (SYNERGY_BUNDLE_DIR ${CMAKE_BINARY_DIR}/bundle)",
-              "set (SYNERGY_BUNDLE_DIR ${CMAKE_INSTALL_PREFIX}/bundle)"
+    inreplace "cmake/Packaging.cmake",
+              "set(SYNERGY_BUNDLE_DIR ${CMAKE_BINARY_DIR}/bundle)",
+              "set(SYNERGY_BUNDLE_DIR ${CMAKE_INSTALL_PREFIX}/bundle)"
+    if OS.linux?
+      # Get rid of Synergy's hardcoded installation path on GNU/Linux.
+      inreplace "cmake/Packaging.cmake",
+                "set(CMAKE_INSTALL_PREFIX /usr)",
+                ""
+    end
```

Comment:
```suggestion
    inreplace "cmake/Packaging.cmake" do |s|
      s.gsub! "set(SYNERGY_BUNDLE_DIR ${CMAKE_BINARY_DIR}/bundle)",
              "set(SYNERGY_BUNDLE_DIR ${CMAKE_INSTALL_PREFIX}/bundle)"
      s.gsub! "set(CMAKE_INSTALL_PREFIX /usr)", "" if OS.linux?
    end
```
Though we could probably just get away with doing
```ruby
prefix.install buildpath/"build/bundle"
```
in lieu of the first `inreplace`.

Reply:
Thank you. I have implemented these changes.